src/app/components/utility/side-bar/side-bar.component.scss

```diff
@@ -2,7 +2,7 @@
 .ev-sidebar {
   display: block;
   height: 100vh;
-  width: 220px;
+  width: 270px;
```

Comment:
![Screenshot from 2020-02-20 20-11-02](https://user-images.githubusercontent.com/32524438/74945565-f9cd7d80-541d-11ea-8353-4c22e9e1a72b.png)


Why we need this file, `header-static` and `footer` file changes? Due to these changes UI doesn't look same on my machine as in the current version.

Reply:
> ![Screenshot from 2020-02-20 20-11-02](https://user-images.githubusercontent.com/32524438/74945565-f9cd7d80-541d-11ea-8353-4c22e9e1a72b.png)
> 
> Why we need this file, `header-static` and `footer` file changes? Due to these changes UI doesn't look same on my machine as in the current version.

The side-nav and top-nav width and look were not same as Eval AI,so i edited it in order to make it look like that of EvalAI. If not required, i will remove them, please let me know.

Reply:
Please show the differences from the current version if there are any cause it looks good on my device without these changes.

Reply:
> Please show the differences from the current version if there are any cause it looks good on my device without these changes.

@Sanji515 Please take a look at the attached screenshot and if previous UI looks better, please let me know.I will reverse these changes then.

Reply:
The only difference which I notice is that there is no vertical scroll on the sidebar right? What's your screen resolution?

Reply:
> The only difference which I notice is that there is no vertical scroll on the sidebar right? What's your screen resolution?

And also the width of the Top-nav bar including the size of the button and texts. My Screen Resolution is 1366 x 768

Reply:
Please remove these changes.

Reply:
> Please remove these changes.

okay @Sanji515